test/integration/api/projects.js

```diff
@@ -1180,3 +1180,51 @@ describe('api: /projects', () => {
   });
 });
 
+describe('api: /projects?forms=true', () => {
```

Comment:
How about a test where there are multiple projects, each with at least one form, to verify that each form is put on the right project?